block/manager.go

```diff
@@ -720,7 +708,7 @@ func (m *Manager) submitNextBatch(ctx context.Context) {
 	// Submit batch to SL
 	// TODO(omritoptix): Handle a case where the SL submission fails due to syncTarget out of sync with the latestHeight in the SL.
 	// In that case we'll want to update the syncTarget before returning.
-	m.submitBatchToSL(nextBatch, resultSubmitToDA)
+	go m.settlementClient.SubmitBatch(nextBatch, m.dalc.GetClientType(), resultSubmitToDA)
```

Comment:
U have a different style with the DA layer and the SL layer
The DA layer has `m.submitBatchToDA` which manages the retries and the result
While now the SL layer is managed entirely from `m.settlementClient`

Reply:
true. that's the [next task ](https://github.com/dymensionxyz/dymint/issues/314) to move the DA submission logic there.